resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -147,6 +147,15 @@ private[spark] object Config extends Logging {
   val CLIENT_CERT_FILE_CONF_SUFFIX = "clientCertFile"
   val CA_CERT_FILE_CONF_SUFFIX = "caCertFile"
 
+
+  val KUBERNETES_CLIENT_OAUTH_TOKEN_PROVIDER =
+    ConfigBuilder("spark.kubernetes.client.oauth.token.provider.class")
+      .doc("A class that implements fabric's OAuthTokenProvider interface to " +
+        "provide a token refresh for long running jobs.")
+      .version("3.1.3")
```

Comment:
Could you use `3.3.0` because the master branch is `3.3.0-SNAPSHOT`?

Reply:
Thank you for the review. Changed to 3.3.0